.travis.yml

```diff
@@ -17,15 +17,28 @@ install:
   - buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID go-get -- go get -u github.com/haya14busa/goverage
   - buildevents step $TRAVIS_BUILD_ID $STEP_SPAN_ID $STEP_START install
 
-script:
-  - STEP_START=$(date +%s)
-  - STEP_SPAN_ID=$(echo script | sum | cut -f 1 -d \ )
-  - buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID codecov-validate -- curl --data-binary @codecov.yml https://codecov.io/validate
-  - buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID make-test-coverage -- make test-coverage
-  # Disable snyk until https://github.com/snyk/snyk/issues/354 is resolved
-  # - if [[ ! -z "${SNYK_TOKEN}" ]]; then snyk monitor --org=czi; snyk test; fi
-  - buildevents step $TRAVIS_BUILD_ID $STEP_SPAN_ID $STEP_START script
-
+jobs:
+  include:
+    - stage: check
+      script:
+        - STEP_START=$(date +%s)
+        - STEP_SPAN_ID=$(echo script | sum | cut -f 1 -d \ )
+        - buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID codecov-validate -- curl --data-binary @codecov.yml https://codecov.io/validate
+        - buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID make-test-coverage -- make test-coverage
+        # Disable snyk until https://github.com/snyk/snyk/issues/354 is resolved
+        # - if [[ ! -z "${SNYK_TOKEN}" ]]; then snyk monitor --org=czi; snyk test; fi
+        - buildevents step $TRAVIS_BUILD_ID $STEP_SPAN_ID $STEP_START script
+    - stage: check
+      script: |
+        set -e
+        export GO111MODULE=on
```

Comment:
shouldn't we set this globally?

Reply:
Hmm yeah, I tried that and something else broke. Let me try again.

Reply:
Not a big deal, don't spend too much time on it